src/Tizen.NUI.Components/Controls/ScrollableBase.cs

```diff
@@ -54,6 +54,57 @@ public Position Position
         }
     }
 
+    /// <summary>
+    /// ScrollOutofBoundEventArgs is to record scroll out-of-bound event arguments which will be sent to user.
+    /// </summary>
+    /// <since_tizen> 8 </since_tizen>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class ScrollOutofBoundEventArgs : EventArgs
```

Comment:
I think 'ScorllOutOfBound' is the correct notation.
In NUI or dotnet, 'O' is capitalized.
->IndexOutOfRangeException / ArgumentOutOfRangeException

Reply:
Done